test/com/google/enterprise/adaptor/database/ResponseGeneratorTest.java

```diff
@@ -247,14 +252,11 @@ public void testRowToText_specialCharacters() throws Exception {
     Response response = newProxyInstance(Response.class, bar);
     Map<String, String> cfg = new TreeMap<String, String>();
     ResponseGenerator resgen = ResponseGenerator.rowToText(cfg);
-
     ResultSet rs = executeQueryAndNext("select * from data");
     resgen.generateResponse(rs, response);
-    String golden =
-        (is(MYSQL) ? "data,data,data\nid,name,quote\n"
-                   :  "DATA,DATA,DATA\nID,NAME,QUOTE\n")
+    String golden = "id,name,quote\n"
```

Comment:
Why did you lowercase the column names? This seems like it would now fail under H2, at least.